File: Python/Easy/1935_Maximum_Number_of_Words_You_Can_Type/solution.py
class Solution:
    def canBeTypedWords(self, text: str, brokenLetters: str) -> int:
        textArray = text.split()
        letterSet = set()
        
        for letter in brokenLetters:
            letterSet.add(letter)

        count = 0
        for t in textArray:
            for w in t:
                if w in letterSet:
                    count -= 1
                    break
            count += 1
        return count
            

File: Golang/Easy/1935_Maximum_Number_of_Words_You_Can_Type/solution.go
package maximumnumberofwordsyoucantype

import "strings"

func canBeTypedWords(text string, brokenLetters string) int {
	letterMap := make(map[rune]bool)

	for _, letter := range brokenLetters {
		letterMap[letter] = true
	}

	textArray := strings.Fields(text)

	count := 0
	for _, t := range textArray {
		for _, w := range t {
			if _, exist := letterMap[w]; exist {
				count -= 1
				break
			}
		}
		count += 1
	}
	return count
}
